source/MaterialXGenShader/TypeDesc.h

```diff
@@ -60,15 +61,15 @@ class MX_GENSHADER_API TypeDesc
     };
 
     /// Empty constructor.
-    constexpr TypeDesc() noexcept :
-        _id(0), _basetype(BASETYPE_NONE), _semantic(SEMANTIC_NONE), _size(0) { }
+    constexpr TypeDesc() noexcept : _id(0), _basetype(BASETYPE_NONE), _semantic(SEMANTIC_NONE), _size(0), _structIndex(0) {}
```

Comment:
For consistency, I'd recommend placing these initializers on their own line, either using one line for each variable or a single line for all variables.